pytest_fixtures/component/rh_cloud.py

```diff
@@ -40,6 +40,8 @@ def rhcloud_registered_hosts(
             org=rhcloud_manifest_org.label,
             rhel_distro=f"rhel{vm.os_version.major}",
         )
+        # workaround to get updated facts for infrastructure_type
```

Comment:
Do you think it's a bug and should be filed ? if yes, once filed put this statement under BZ is open condition.

Reply:
It turns out to be a bug, filed https://issues.redhat.com/browse/SAT-22661
Closing this PR.